docs/config-files/config.py

```diff
@@ -0,0 +1,14 @@
+#This file contains all of the user based inputs
```

Comment:
I think a config file should not be in docs but should be with the main code.  Also, I wonder if it is better to make this a yaml file and make it look more like a dictionary.